multicluster/controllers/multicluster/member/clusterset_controller.go

```diff
@@ -101,6 +101,10 @@ func (r *MemberClusterSetReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		if !apierrors.IsNotFound(err) {
 			return ctrl.Result{}, err
 		}
+		if r.clusterSetID != common.ClusterSetID(req.Name) {
```

Comment:
Does the namespace of ClusterSet matter? From the validation code, I see it's not allowed to have multiple ClusterSets created in a Namespace but it's allowed to have multiple ClusterSets in different Namespaces.

Reply:
ClusterSet Namespace is a config option of Multi-cluster Controller, and we only look at ClusterSets in that Namespace.

Reply:
Yes, we have namespace selector when building up the cache in controller.go for ClusterSet, maybe we can remove the namespace check in the webhook, I will double check.

Reply:
Got it, thanks for explanation.